docs/rules/no-implicit-globals.md

```diff
@@ -0,0 +1,40 @@
+# disallow `var` and named functions in global scope (no-implicit-globals)
+
```

Comment:
No, only `var` creates an implicit global at the top-level. It was right the first time.